### DISCUSSION TITLE
Phase 7 is now finished.

### DISCUSSION THREAD

**Discussion by daveajones** [Podcastindex-org/podcast-namespace#665] [Announcements - Changes Made]
Phase 7 is now formally finished and we can re-focus on Phase 8.  My strong recommendation for Phase 8 and beyond is that we focus on a smaller set of changes in order to move faster.

**Comment by samsethi** [comment]
Hi Dave 

Could I kick off Phase 8 by asking for Podcast [Pinning Tag](https://github.com/Podcastindex-org/podcast-namespace/discussions/570) - this is a tag I recommended ages ago.   It allows podcasters to feature their trailer or another episode at the top of their feed. I proposed this tag because Spotify offer this feature and I wanted feature parity in P2.0.  

It already works in TrueFans. Currently, we pin the trailer episode by default but allow podcasters to pick another episode to be their default/lead in episode via their admin dashboard.  What would be great is if the podcast pinning episode was in the feed so we could just read it.  

<img width="1208" alt="Cursor_and_TrueFans___Shows___Podcasting_2_0" src="https://github.com/user-attachments/assets/b8abdffb-69b0-47f3-a9e8-887b1aa49c04">

The other tag I would love to propose is the <podcast:activity> tag

**Comment by jamescridland** [comment]
I'd like to see if we can extend the `podcast:location` tag so that it can contain details about where a podcast was recorded, as well as (or instead of) where a podcast is about. Would there be any interest in doing this?

**Reply by samsethi** [reply]
Hi James 

I would love to extend the location tag to include both the recording location and the subject location.  Daniel J Lewis made a first stab at this some months ago. https://github.com/Podcastindex-org/podcast-namespace/discussions/616#discussioncomment-8684699 

His idea is simple and doesn't change the tag other than adding a few new attributes.  TrueFans already has the location tag linked to Google Maps and we would like to build a feature to show all the podcasters on a map based on their location.  I think small meetups or podcamps could be organised by neighbouring podcasts.  I think maps.fm tried this but as a standalone app??? 

<img width="1410" alt="Fullscreen_06_10_2024__22_41" src="https://github.com/user-attachments/assets/e3b8cc1b-7b02-41c2-9f9e-f811b3a13e21">



Benjamin Bellamy also made a strong case for a country tag which has a different use case to the one for the location tag. 
https://github.com/Podcastindex-org/podcast-namespace/discussions/616#discussioncomment-8697271

TrueFans already has support for the country tag which should be part of Phase 8 along with the extended location tag.  

https://github.com/Podcastindex-org/podcast-namespace/discussions/616#discussioncomment-8697271 

**Phase 8:** ideas so far? 

- Podcast Image extension 
- Podcast Pinning.
- Location - extended the current tag to include the recording location 
- Content Links - 
- Funding Tag - extended to include an item-level funding tag 
- Medium=Events - https://github.com/Podcastindex-org/podcast-namespace/discussions/640 
- Country Tag - https://github.com/Podcastindex-org/podcast-namespace/discussions/616#discussion-6261203 
- PodID - aka PersonRoll - this shows the list of shows I host and other shows I was a guest. This will be in the form of the podroll and publisher feeds. 

/cc @daveajones

**Comment by daveajones** [comment]
Great to see these suggestions.  Thanks guys.
